### ISSUE TITLE
Mastodon not working after update?

### ISSUE THREAD

**Issue by Alkarex** [friendica/friendica#4673]
I have just updated from develop 06d6d6c758b26713f97e3f9929f39e496171a023 to the current develop.
Everything used to work just fine, and now I do not receive Mastodon news, only GNU Social and Friendica.
Any idea what that could be due to, and how to fix it?

There is apparently no error in my logs (everything is in NORMAL). I have just increased the verbosity to TRACE.

Maybe relevant:

```
# grep -i masto friendica.log
2018-03-24T10:50:06Z@wrk5ab62d5b8910f4.98908181 [NORMAL]:DiscoverPoCo.php:190:discoverUsers     Check profile https://mastodon.hackerlab.fr/@Saberwarthog
2018-03-24T10:50:06Z@log5ab62d5e5bac38.23583270 [NORMAL]:Worker.php:318:execFunction    Process 28008 - Prio 40 - ID 929282: DiscoverPoCo ["DiscoverPoCo","check_profile","https:\/\/mastodon.hackerlab.fr\/@Saberwarthog"] - Process PID: wrk5ab62d5e95ddc4.42953428
2018-03-24T13:09:11Z@log5ab64df71575a7.25806323 [NORMAL]:Item.php:613:insert    item parent https://mastodon.xyz/users/Zergy/statuses/99738281998732378 for 1 was not found - ignoring item
2018-03-24T14:54:54Z@log5ab666bdeee819.82289240 [NORMAL]:Item.php:613:insert    item parent https://mastodon.social/users/thewaether/statuses/99739237244464176 for 1 was not found - ignoring item
2018-03-24T15:01:03Z@log5ab6682b8ac8f1.41983948 [NORMAL]:Item.php:613:insert    item parent https://mastodon.social/users/thrrgilag/statuses/99739459170595928 for 1 was not found - ignoring item
2018-03-24T15:40:03Z@wrk5ab67153bae603.06685677 [NORMAL]:DiscoverPoCo.php:190:discoverUsers     Check profile https://mastodon.xyz/@athanael
2018-03-24T15:40:04Z@log5ab671544a8209.16494928 [NORMAL]:Worker.php:318:execFunction    Process 22339 - Prio 40 - ID 929532: DiscoverPoCo ["DiscoverPoCo","check_profile","https:\/\/mastodon.xyz\/@athanael"] - Process PID: wrk5ab671548767d3.62955869
2018-03-24T16:43:51Z@log5ab68046dc6682.50706527 [NORMAL]:Item.php:613:insert    item parent https://mastodon.xyz/users/Zergy/statuses/99739487811410182 for 1 was not found - ignoring item
```

**Comment by annando** [comment]
Could you - for a test - switch to master?

**Comment by Alkarex** [comment]
No entry with `dba.php`:

```sh
grep -i dba.php friendica.log
```

I will try to switch to master, but I am a bit worried to do so, since there has been many changes in file location, DB, addons, etc.

**Comment by Alkarex** [comment]
@annando I have switched to master branch, re-run cron. No apparent error (neither in friendica.log not in my Apache logs), but still nothing from Mastodon :-(
Any other idea?

**Comment by Alkarex** [comment]
For the record, my instance is https://f.a80.fr/profile/alkarex

**Comment by annando** [comment]
Just execute manually the database update via shell. And the path to the worker is under /scripts/worker.php.

**Comment by Alkarex** [comment]
```sh
# php scripts/dbstructure.php update
# sudo -u www-data /usr/bin/php -f scripts/worker.php
# grep -i dba.php friendica.log
```

**Comment by Alkarex** [comment]
I have just run a `/repair_ostatus` (while on master branch) and it looks like it helped. I have received the missing Mastodon update.

**Comment by Alkarex** [comment]
Now I got one line with an error in dba.php, but I am not sure it is related:
```
2018-03-24T18:25:41Z@wrk5ab69825eec6e8.01410424 [NORMAL]:OnePoll.php:29:execute start
2018-03-24T18:25:42Z@wrk5ab69825eec6e8.01410424 [NORMAL]:OnePoll.php:158:execute        poll: (stat-2445) IMPORTER: Alkarex, CONTACT: YunoHost
2018-03-24T18:25:42Z@wrk5ab69825eec6e8.01410424 [NORMAL]:OnePoll.php:583:execute        Consume feed of contact 2445
2018-03-24T18:25:43Z@wrk5ab69825eec6e8.01410424 [NORMAL]:dba.php:545:e  DB Error 1292: Incorrect date value: '0000-00-00' for column 'bd' at row 1
dba::update, GContact::update, OStatus::import, consume_feed, OnePoll::execute, call_user_func_array, Worker::processQueue
UPDATE `contact` SET `bd` = '0000-00-00' WHERE `id` = '207'
2018-03-24T18:25:52Z@wrk5ab69812976ec2.76490566 [NORMAL]:OnePoll.php:605:execute        Contact 16307 returned hub: https://mastodon.partipirate.org/api/push Network: stat Relation: 2 Update: 1
```

**Comment by annando** [comment]
it is not related, I think. But this is some other problem.

**Comment by MrPetovan** [comment]
Closed through #4679
